pkg/app/gen/html.go

```diff
@@ -1284,6 +1284,11 @@ var attrs = map[string]attr{
 	},
 
 	// K:
+	"key-value": {
+		Name: "SetAttribute",
```

Comment:
All the syntax uses a bare name to set its attribute or event handler.
Lets just call this `Attr`